packages/rollup/test/esm_config/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "generated_file_test")
+load("//packages/rollup:index.bzl", "rollup_bundle")
+
+rollup_bundle(
+    name = "esm_config",
+    config_file = "rollup.config.mjs",
+    entry_point = "input.js",
+    sourcemap = "false",
+    supports_workers = True,
+    deps = [
+        # Fake test ESM package from `npm_deps.bzl`.
+        "@npm//esm_pkg",
```

Comment:
Do we need to use an npm package to reproduce the issue? Can't just be plain local imports or a `js_library`?

Reply:
I wasn't able to reproduce actual ESM issues with a local script because rollup attempts to bundle the rollup config as it seems (ironically). I think having such a package is a good thing for other future ESM additions as well.

Reply:
Happy to explore this more, but I believe this should be fine, or do you have any concerns?

Reply:
Would just be nice to avoid polluting the root package.json, although there's already a `testy` package in there so I guess this just follows that same pattern. Maybe just name it `testy_esm` so it's obvious they serve the same purpose?

Reply:
good idea, agreed. btw: there are many other test packages, some named just `hello` 😄